app/assets/javascripts/components/general_button.es6.jsx

```diff
@@ -1,5 +1,10 @@
 class GeneralButton extends React.Component {
 
+  constructor(props) {
+    super(props);
```

Comment:
Good question! Props are going to come from the parent component, not the `React.Component` parent class. The parent component is whatever component decides to render this `GeneralButton` component as a child. In this case the parent component is the `LoginForm` or the `SignupForm` component. Check the pull request description on what the `super` call is about.